core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_interaction.html

```diff
@@ -151,3 +156,12 @@ <h3 ng-if="stateInteractionIdService.displayed && (customizationModalReopened ||
     </div>
   </div>
 </script>
+<style>
+  .oppia-interaction-customization-save-disabled {
```

Comment:
.oppia-save-interaction-button-disabled might be a better name, since you're disabling the "'save interaction' button".